prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-care-types-page/health-auth-care-types-page.component.ts

```diff
@@ -95,7 +104,11 @@ export class HealthAuthCareTypesPageComponent implements OnInit {
     this.healthAuthResource.getHealthAuthorityById(this.route.snapshot.params.haid)
       .subscribe(({ careTypes }: HealthAuthority) =>
         (careTypes?.length)
-          ? careTypes.map(ct => this.addCareType(ct.careType))
+          ? careTypes.map(ct => {
+            const id = ct.id;
+            const careTypeName = ct.careType;
+            this.addCareType({ id, careTypeName });
```

Comment:
Could reduce this to a line
``` this.addCareType({ ct.id, careTypeName: ct.careType })```

Reply:
Reduced, you meant `this.addCareType({ id: ct.id, careTypeName: ct.careType })` right?

Reply:
can't recall but i thought its intuitive and since if you just pass obj.id it infers it as a field with the same key "id"

Reply:
Sadly not when the id is a property of another object